tests/bgp/test_traffic_shift.py

```diff
@@ -184,7 +184,7 @@ def parse_routes_process(node=None, results=None):
             routes[entry] = community
         results[hostname] = routes
 
-    all_routes = parallel_run(parse_routes_process, (), {}, neigh_hosts.values(), timeout=240, concurrent_tasks=8)
+    all_routes = parallel_run(parse_routes_process, (), {}, neigh_hosts.values(), timeout=240, concurrent_tasks=4)
```

Comment:
Jibin, if this is a concurrency issue, why reducing the number of tasks would eliminate the problem?

Reply:
Reducing the number of tasks may decrease the possibility of the issue. When changing it to 4 tasks,  we tested it 20 times, the issue didn't happen again. If we still use 8 tasks, this issue will happen one time about every 3 times.